test/e2e/max_pods.go

```diff
@@ -100,6 +100,12 @@ var _ = Describe("MaxPods", func() {
 		By(fmt.Sprintf("Starting additional %v Pods to fully saturate the cluster and trying to start a new one", podsNeededForSaturation))
 		expectNoError(RunRC(config))
 
+		// Make sure that cluster is saturated
```

Comment:
This is what's happening right now (kind of). What I want to do is to check if failures are really caused by some pods being unhealthy in some way.